backend/src/test/java/de/benjaminranft/spotown/controller/DiscoveryControllerIntegrationTest.java

```diff
@@ -0,0 +1,169 @@
+                                        new ArrayList<>(List.of("food", "art")))
+                        ))
+                ),
+                new User(
+                        "heinz",
+                        passwordHeinz,
+                        new ArrayList<>(List.of(
+                                new Discovery("123", timestamp, "Pizza Place", "Sample Street 3", "https://google.com", "11:00 - 12:00", "04023457596", "https://google.com", "https://google.com", "Sample notes sample notes sample notes",
+                                        new ArrayList<>(List.of("nature", "art"))),
+                                new Discovery("456", timestamp, "Sushi Place", "Sample Street 4", "https://google.com", "11:00 - 12:00", "04023457596", "https://google.com", "https://google.com", "Sample notes sample notes sample notes",
+                                        new ArrayList<>(List.of("drink", "nature")))
+                        ))
+                )
+        ));
+    }
+
+        private String backendAccessLink(){
+            return "http://localhost:" + port + "/api/discoveries";}
+
+        private String login(){
+            ResponseEntity<String> response = testRestTemplate.postForEntity("http://localhost:" + port + "/auth/login", new LoginDto(
+                    "heinz", "password"
+            ), String.class);
+
+            return response.getBody();
+        }
+
+        private <T> HttpEntity<T> getValidAuthorizationEntity(T data){
+            String token =login();
+
+            HttpHeaders headers = new HttpHeaders();
+            headers.setBearerAuth(token);
+            return new HttpEntity<>(data, headers);
+}
+
+@Test
+@DisplayName("The getDiscoveries method should return the correct discovery list")
+public void  getDiscoveriesTest(){
+
+        //GIVEN
+        String url = backendAccessLink();
+
+        //WHEN
+        HttpEntity<Void> entity = getValidAuthorizationEntity(null);
+        ResponseEntity<Discovery[]> response = testRestTemplate.exchange(url, HttpMethod.GET, entity, Discovery[].class);
+
+        List<Discovery> expectedList = new ArrayList<>(List.of(
+                new Discovery("123", Instant.parse("2020-11-18T18:35:24.00Z"), "Pizza Place", "Sample Street 3", "https://google.com", "11:00 - 12:00", "04023457596", "https://google.com", "https://google.com", "Sample notes sample notes sample notes",
+                        new ArrayList<>(List.of("nature", "art"))),
+                new Discovery("456", Instant.parse("2020-11-18T18:35:24.00Z"), "Sushi Place", "Sample Street 4", "https://google.com", "11:00 - 12:00", "04023457596", "https://google.com", "https://google.com", "Sample notes sample notes sample notes",
+                        new ArrayList<>(List.of("drink", "nature")))
+
+        ));
```

Comment:
I think this block is an item from the //given section.

Reply:
We usually add the expected result in the //WHEN or //THEN section of our tests, but I'll check it whether it makes sense to move it up there, thank you!